tests/clients/test_ngsi_v2_cb.py

```diff
@@ -338,6 +340,50 @@ def test_subscriptions(self):
             for sub in subs:
                 client.delete_subscription(subscription_id=sub.id)
 
+    def test_registration_set_status(self):
+        """
+        Test subscription operations of context broker client
+        """
+        with ContextBrokerClient(fiware_header=self.fiware_header) as client:
+            reg_example = {
+                "description": "One registration to catch them all",
+                "dataProvided": {
+                    "entities": [
+                        {
+                            "idPattern": ".*",
+                            "type": "Room"
+                        }
+                    ],
+                    "attrs": [
+                        "temperature"
+                    ],
+                },
+                "provider": {
+                    "http": {
+                        "url": "http://localhost:1234",
+                    },
+                    "supportedForwardingMode": "all"
+                }
+            }
+        reg = Registration(**reg_example)
+        reg_id = client.post_registration(registration=reg)
```

Comment:
Registration is not imported. And I get a parsing error for http

@tstorek are the reworked models already merged? I am not sure

Reply:
@dnikolay-ebc I added the new models to the development. Please try to merge the development into your branch